lib/Dialect/TTNN/Analysis/LegalLayoutAnalysis.cpp

```diff
@@ -0,0 +1,293 @@
+// SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Can this be marked as rename instead of delete+add so that only new code can be highlighted?

Reply:
I think that this is a git / github internal decision and that there is not way to force it. Do you know a way?

Reply:
This might work https://jtemporal.com/renaming-files-in-git-the-right-way/

Reply:
Didn't know this, ty @nobradovictt

Reply:
Tried this, did work on single commit, but shown the same on the whole PR. Might be because there are a lot of changes in the file.